parse.js

```diff
@@ -8,16 +8,20 @@
 
 const ARGUMENT_SEPARATION_REGEX = /([^=\s]+)=?\s*(.*)/;
 
-function Parse(argv) {
+const Parse = (argv) => {
   // Removing node/bin and called script name
   argv = argv.slice(2);
 
   const parsedArgs = {};
   let argName, argValue;
 
-  argv.forEach(function (arg) {
+  for (let arg of argv) {
```

Comment:
Any particular reason why you've changed the functional foreach to a regular foreach?

Reply:
In foreach loop if we use return then it will only return for that argument (arg in loop) not return from function. In our case if syntax is wrong then we have to return from function and in foreqch loop we can't do that.

Reply:
And there is not required function so it is easy and efficient.

Reply:
Ah i see! My bad, haven't noticed that. 
But now I'm thinking about the way you handle the error. You now immediately return an empty object and just log the error to the console. Depending on the client code this could invoke some default logic, which might not be intended, cause other 'correctly' parsed arguments are not set. (Hope you get what i mean)
Also client code cannot handle the error. Right now I can think of two possible ways to fix it: either throw an error and let client code handle it (showing an error message, stopping execution or whatever) or using a callback function on error, which allows for handling it and maybe continuing the parsing (if no cb is set your error message could be shown).

Reply:
I just think about client's code would not break and just show error message. But, yes you are right that can be possible we should stop execution so that can prevent unwanted code execution.

Reply:
@NiclasThobaben  Just updated the PR, throw error if regex not match. not did` callback function on error, which allows for handling it and maybe continuing the parsing` because if we do continue parsing the argument then it will give different output then expected. like` node script.js -m = message` it will give` {m: true, message: true}` which is not expected.

Reply:
@NiclasThobaben @eveningkid is this looks good? Or any changes. I am happy if you guys have any suggestion.